projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-viewer.component.ts

```diff
@@ -133,7 +127,11 @@ export class PdfViewerComponent implements AfterContentInit, OnChanges, OnDestro
       this.clearAnnotationSet();
       this.documentId = this.extractDMStoreDocId(this.url);
       if (this.enableRedactions) {
-        this.store.dispatch(new fromRedactionActions.LoadRedactions(this.documentId));
+        this.toolbarEvents.redactionMode.subscribe(redaMode => {
+          if (redaMode) {
```

Comment:
this is just my preference, so it's up to you if you want to change it, but maybe call it redactMode instead of redaMode